velox/docs/functions/spark/aggregate.rst

```diff
@@ -73,12 +73,50 @@ General Aggregate Functions
 
     Returns the value of `x` associated with the maximum value of `y`.
     Note: Spark provides a non-strictly comparator which is greater than or equals to.
-    Eg. SELECT max_by(x, y) FROM VALUES (('a', 10)), (('b', 50)), (('c', 50)) AS tab(x, y);
-        > c
+
+    Example::
+
+        SELECT max_by(x, y)
+        FROM (
+            VALUES
+                ('a', 10),
+                ('b', 50),
+                ('c', 50)
+        ) AS t(x, y);
+
+    Returns c
 
 .. spark:function:: min_by(x, y) -> x
 
     Returns the value of `x` associated with the minimum value of `y`.
     Note: Spark provides a non-strictly comparator which is less than or equals to.
-    Eg. SELECT min_by(x, y) FROM VALUES (('a', 10)), (('b', 10)), (('c', 50)) AS tab(x, y);
-        > b
\ No newline at end of file
+
+    Example::
+
+        SELECT min_by(x, y)
+        FROM (
+            VALUES
+                ('a', 10),
+                ('b', 10),
+                ('c', 50)
+        ) AS t(x, y);
+
+    Returns b
+
+.. spark:function:: sum(x) -> x
```

Comment:
Is this signature correct? What is return type of sum(TINYINT)? Is it TINYINT or BIGINT? If TINYINT, then how do we handle overflow?

Reply:
Apologies for the confusion. I've made updates based on the reference: https://facebookincubator.github.io/velox/functions/presto/aggregate.html#avg

For other integer types, the return type is `bigint`, so there won't be overflow.

Reply:
Got it. Thank you for clarifying.